src/registrar/config/settings.py

```diff
@@ -475,6 +475,10 @@ class JsonServerFormatter(ServerFormatter):
 
     def format(self, record):
         formatted_record = super().format(record)
+
+        if not hasattr(record, "server_time"):
+            record.server_time = self.formatTime(record, self.datefmt)
```

Comment:
This was a minor fix to an edge case where this attribute doesn't exist, causing log formatting to revert back to its old "one log per \n" behavior. I can include it as a separate PR if we want, but thought it was small enough to sneak in.